### ISSUE TITLE
/opt/jython2.7.0/bin/pip install requests not working, SSL handshake error...

### ISSUE THREAD

**Issue by alxgrtnstrngl** [jythontools/jython#20]
```
[root@lab ~]# /opt/jython2.7.0/bin/pip install requests
Downloading/unpacking requests
Got this failure javax.net.ssl.SSLException: handshake timed out during SSL handshake (<_realsocket at 0x216 type=client open_count=1 channel=[id: 0xd2854932, /192.168.1.10:60386 => pypi.python.org/23.235.46.223:443] timeout=15.0>)
Got this failure javax.net.ssl.SSLException: handshake timed out during SSL handshake (<_realsocket at 0x217 type=client open_count=1 channel=[id: 0x4203ec5f, /192.168.1.10:54679 => pypi.python.org/23.235.46.223:443] timeout=15.0>)
  Cannot fetch index base URL https://pypi.python.org/simple/
Got this failure javax.net.ssl.SSLException: handshake timed out during SSL handshake (<_realsocket at 0x218 type=client open_count=1 channel=[id: 0xd41cc95e, 0.0.0.0/0.0.0.0:43107 :> pypi.python.org/23.235.46.223:443] timeout=15.0>)
  Could not find any downloads that satisfy the requirement requests
Cleaning up...
No distributions at all found for requests
Storing debug log for failure in /root/.pip/pip.log
```

**Comment by anirban1c** [comment]
have noticed these in the pip log as well when using pip behind a corporate proxy

/vagrant/jython/27/bin/pip run on Tue May 12 10:59:45 2015
Downloading/unpacking clamp
  Getting page https://pypi.python.org/simple/clamp/
  Could not fetch URL https://pypi.python.org/simple/clamp/: connection error: hostname 
'localhost' doesn't match either of 'www.python.org', 'python.org', 'pypi.python.org', '
docs.python.org', 'testpypi.python.org', 'bugs.python.org', 'wiki.python.org', 'hg.pytho
n.org', 'mail.python.org', 'packaging.python.org', 'pythonhosted.org', 'www.pythonhosted
.org', 'test.pythonhosted.org', 'us.pycon.org', 'id.python.org'
  Will skip URL https://pypi.python.org/simple/clamp/ when looking for download links fo
r clamp
  Getting page https://pypi.python.org/simple/
  Could not fetch URL https://pypi.python.org/simple/: connection error: hostname 'local
host' doesn't match either of 'www.python.org', 'python.org', 'pypi.python.org', 'docs.p
ython.org', 'testpypi.python.org', 'bugs.python.org', 'wiki.python.org', 'hg.python.org'
, 'mail.python.org', 'packaging.python.org', 'pythonhosted.org', 'www.pythonhosted.org', 'test.pythonhosted.org', 'us.pycon.org', 'id.python.org'
  Will skip URL https://pypi.python.org/simple/ when looking for download links for clamp
  Cannot fetch index base URL https://pypi.python.org/simple/
  URLs to search for versions for clamp:
  * https://pypi.python.org/simple/clamp/
  Getting page https://pypi.python.org/simple/clamp/
  Could not fetch URL https://pypi.python.org/simple/clamp/: connection error: hostname 'localhost' doesn't match either of 'www.python.org', 'python.org', 'pypi.python.org', 'docs.python.org', 'testpypi.python.org', 'bugs.python.org', 'wiki.python.org', 'hg.python.org', 'mail.python.org', 'packaging.python.org', 'pythonhosted.org', 'www.pythonhosted.org', 'test.pythonhosted.org', 'us.pycon.org', 'id.python.org'
  Will skip URL https://pypi.python.org/simple/clamp/ when looking for download links for clamp

**Comment by anirban1c** [comment]
where env is set with

http_proxy=http://localhost:8008
https_proxy=http://localhost:8008
no_proxy=localhost,127.0.0.0/8

**Comment by trixpan** [comment]
+1 totally broken at the moment. puthon 2.6, python2.7 verisons of pip work without issues

**Comment by darjus** [comment]
Fixed since 1.7.1b2. Please try it out.
